qemu/tests/timedrift_check_when_crash.py

```diff
@@ -29,6 +29,7 @@ def run(test, params, env):
     nmi_cmd = params.get("nmi_cmd", "inject-nmi")
     sleep_time = float(params.get("sleep_time", 1800))
     deviation = float(params.get("deviation", 5))
+    os_type = params.get("os_type", "linux")
```

Comment:
I think it's not a good idea to set os_type default value to "linux".

Reply:
Updated.